src/mbed_os_tools/test/host_tests_plugins/module_reset_jlink.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2018, Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+
+from .host_test_plugins import HostTestPluginBase
+
+
+RESET_FILE = "reset.jlink"
+
+class HostTestPluginResetMethod_Jlink(HostTestPluginBase):
+
+    # Plugin interface
+    name = 'HostTestPluginResetMethod_Jlink'
+    type = 'ResetMethod'
+    capabilities = ['JTAG', 'SWD']
+    required_parameters = ['mcu']
+
+    def is_os_supported(self, os_name=None):
+        """! In this implementation this plugin only is supporeted under Windows machines
```

Comment:
Little spelling error here:
```suggestion
        """! In this implementation this plugin only is supported under Windows machines
```